src/components/docs-product-logo.js

```diff
@@ -1,13 +1,22 @@
 import React from "react"
 import getCloudflareDocsConfig from "../utils/get-cloudflare-docs-config"
+import getUniqueReadableID from "../utils/get-unique-readable-id"
 import AccessibleSVG from "./accessible-svg"
 
+const titleID = getUniqueReadableID("title")
+
 export default () => {
-  const { product, productLogoPathD }  = getCloudflareDocsConfig()
+  const { product, productLogoPathD, productIconKey, logoSVGContent } = getCloudflareDocsConfig()
 
-  return (
+  return logoSVGContent ? (
```

Comment:
Maybe check _both_ `productIconKey` and `logoSVGContent` truthiness since both are used in this path

Reply:
there might be instances where some products will not have a productIconKey and the productIconKey is only used for the title of the span and if it doesnt exist we put a generic logo text there